slider.css

```diff
@@ -0,0 +1,57 @@
+.slider {
+  display: block;
+}
+
+.slider-container {
+  display: inline-block;
+  position: relative;
+}
+
+.image {
+  object-fit: cover;
+  object-position: 50% 50%;
+}
+.slider-list {
+  padding: 0;
+  margin: 0;
+}
+
+.slider-item {
+  display: none;
+}
+
+.slider-item.active {
```

Comment:
It's better to have .slider-item-active class. .active class itself can be used in many places and it's difficult to understand it without a knowing of a context. 

Minus unnecessary styles cascade